docs/reference/objects/package/step/item.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Package Step Item
+parent: Objects
```

Comment:
Where is this drop accessed from? is it package_step.items? It doesn't look like this page is referenced from anywhere. (There is a reference to items within package_product but it looks like that is linking to package.items instead)

Reply:
I've fixed 👍